README.md

```diff
@@ -1,56 +1,73 @@
-# gulp-mdast
+# gulp-remark
 
 [![NPM version][npm-image]][npm-url]
 [![Build Status][travis-image]][travis-url]
 [![Coveralls Status][coveralls-image]][coveralls-url]
 [![Dependency Status][depstat-image]][depstat-url]
 
-> Gulp plugin for Mdast
+> Gulp plugin for [Remark][remark] — markdown processor powered by plugins
 
 ## Install
 
-    npm install --save gulp-mdast
+    npm install --save-dev gulp-remark
 
 ## Usage
 
 ```js
-import gulpMdast from 'gulp-mdast';
-
-gulpMdast('unicorns'); // unicorns
+import gulp from 'gulp';
+import remark from 'gulp-remark';
+import html from 'remark-html';
+import lint from 'remark-html';
+
+gulp.task('default', () =>
+  gulp.src('*.md')
+    .pipe(remark().use(html).use(lint))
```

Comment:
For what? I would be good as example, but personally I think npm scripts do this job well enough.